README.md

```diff
@@ -2,6 +2,18 @@
 
 The purpose of this repo is to aggregate issues filed against the [kubeadm component](https://github.com/kubernetes/kubernetes/tree/master/cmd/kubeadm).
 
+## What is Kubeadm ?
+Kubeadm is a tool built to provide best-practice "fast paths" for creating Kubernetes clusters.
+It performs the actions necessary to get a minimum viable, secure cluster up and running in a user friendly way.
+Kubeadm's scope is limited to the local node filesystem and the Kubernetes API, and it is intended to be a composable building block of higher level tools. 
+
+
+## Common Kubeadm cmdlets 
+1. **kubeadm init** to bootstrap kubernetes control-plane nodes.
+1. **kubeadm join** to bootstrap a Kubernetes worker node and join it to the cluster.
```

Comment:
`1. **kubeadm join** to bootstrap a Kubernetes worker node or an additional control plane node, and join it to the cluster.`